### PR TITLE
JBPM-7286 - TaskVariableImpl table is not maintained by LogCleanupCom…

### PR DESCRIPTION
**Pull request by mswiderski**
…mand

depends on https://github.com/kiegroup/droolsjbpm-knowledge/pull/307

### CODE REVIEW

**Comment by mswiderski** [comment]
Jenkins retest this

**Comment by mswiderski** [comment]
Jenkins retest this

File: jbpm-human-task/jbpm-human-task-audit/src/main/java/org/jbpm/services/task/audit/service/TaskVariableDeleteBuilderImpl.java [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+		addObjectParameter(TASK_VARIABLE_DATE_ID_LIST, "created on date", ensureDateNotTimestamp(date));
+		return this;
+	}
+
+	@Override
+	public TaskVariableDeleteBuilder dateRangeStart(Date rangeStart) {
+		if (checkIfNull(rangeStart)) {
+			return this;
+		}
+		addRangeParameter(TASK_VARIABLE_DATE_ID_LIST, "created on date range end", ensureDateNotTimestamp(rangeStart)[0], true);
```
Comment (MarianMacik) [inline jbpm-human-task/jbpm-human-task-audit/src/main/java/org/jbpm/services/task/audit/service/TaskVariableDeleteBuilderImpl.java]: Should be "created on date range start" I think.

File: jbpm-human-task/jbpm-human-task-audit/src/main/java/org/jbpm/services/task/audit/service/TaskVariableDeleteBuilderImpl.java [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    public TaskVariableDeleteBuilderImpl(CommandExecutor cmdExecutor ) {
+        super(cmdExecutor);
+        intersect();
+    }
+  
+    public TaskVariableDeleteBuilderImpl(JPAAuditLogService jpaAuditService) { 
+       super(jpaAuditService);
+       intersect();
+    }
+
```
Comment (MarianMacik) [inline jbpm-human-task/jbpm-human-task-audit/src/main/java/org/jbpm/services/task/audit/service/TaskVariableDeleteBuilderImpl.java]: Formatting is a little bit off here.

**Comment by mswiderski** [comment]
@MarianMacik changes applied

**Comment by mswiderski** [comment]
Jenkins retest this

**Pull request merged**